internal/doc/butane.yaml

```diff
@@ -326,6 +326,15 @@ root:
         - name: luks
```

Comment:
The `desc` line just above this one needs to be updated to mention the new allowed layouts.

Reply:
Hi @jlebon , 
The moment I add the layouts ```desc: the disk layout of the target OS image. Supported values are `aarch64`, `ppc64le`, `s390x-eckd`, `s390x-virt`, `s390x-zfcp` and `x86_64`. Defaults to `x86_64`. ```  when I run `./generate`  It generates the docs for the version lower to the 1.6_exp  like below.

 It must be only for v1_6-exp.md. and openshift-v4_15-exp.md, right?

	 modified:   docs/config-fcos-v1_3.md
	modified:   docs/config-fcos-v1_4.md
	modified:   docs/config-fcos-v1_5.md
	modified:   docs/config-fcos-v1_6-exp.md
	modified:   docs/config-openshift-v4_10.md
	modified:   docs/config-openshift-v4_11.md
	modified:   docs/config-openshift-v4_12.md
	modified:   docs/config-openshift-v4_13.md
	modified:   docs/config-openshift-v4_14.md
	modified:   docs/config-openshift-v4_15-exp.md
	modified:   docs/config-openshift-v4_8.md
	modified:   docs/config-openshift-v4_9.md

Reply:
Ahh yes, good point.

I think we can add a transform rule so that the older specs remain unaffected. See the `transforms` knob used in other parts of this file.